docs/root/configuration/http/http_filters/lua_filter.rst

```diff
@@ -63,6 +59,81 @@ Configuration
 * :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.lua.v3.Lua>`
 * This filter should be configured with the name *envoy.filters.http.lua*.
 
+A simple example of configuring Lua HTTP filter that contains only :ref:`inline_code 
+<envoy_v3_api_field_extensions.filters.http.lua.v3.Lua.inline_code>` is as follow:
+
+.. code-block:: yaml
+
+  name: envoy.filters.http.lua
+  typed_config:
+    "@type": type.googleapis.com/envoy.extensions.filters.http.lua.v3.lua
+    inline_code: |
+      -- Called on the request path.
+      function envoy_on_request(request_handle)
+        -- Do something.
+      end
+      -- Called on the response path.
+      function envoy_on_response(response_handle)
+        -- Do something.
+      end
+
+By default, Lua script defined in `inline_code` will be treated as a `GLOBAL` script. Envoy will 
```

Comment:
A reminder that in our .rst renderer, 

```
`GLOBAL`
```

won't be wrapped inside `<code class="docutils literal notranslate"><span class="pre">GLOBAL</span></code>` element. (you can try to run the `./docs/build.sh` and check how this will be rendered, the resulting htmls will be in `generated/docs/` dir).

Use double backticks if you want it to be rendered as: 

![image](https://user-images.githubusercontent.com/73152/85939054-9360ea80-b93c-11ea-94dc-d4881c9098ae.png)

```
``GLOBAL``
```

Reply:
Thanks. That's what i really want.